python/cudf/cudf/core/series.py

```diff
@@ -671,6 +671,48 @@ def axes(self):
         """
         return [self.index]
 
+    @property  # type: ignore
+    @_cudf_nvtx_annotate
+    def hasnans(self):
+        """
+        Return True if there are any NaNs or nulls.
+
+        Returns
+        -------
+        out : bool
+            If Series has at least one NaN or null value, return True,
+            if not return False.
+
+        Examples
+        --------
+        >>> import cudf
+        >>> import numpy as np
+        >>> series = cudf.Series([1, 2, np.nan, 3, 4], nan_as_null=False)
+        >>> series
+        0    1.0
+        1    2.0
+        2    NaN
+        3    3.0
+        4    4.0
+        dtype: float64
+        >>> series.hasnans
+        True
+
+        `hasnans` returns `True` for the presence of any `NA` values:
+
+        >>> series = cudf.Series([1, 2, 3, None, 4])
+        >>> series
+        0       1
+        1       2
+        2       3
+        3    <NA>
+        4       4
+        dtype: int64
+        >>> series.hasnans
+        True
+        """
+        return self._column.has_nulls(include_nan=True)
```

Comment:
Why there is calling `has_nulls` in the `has_nans` function?

Reply:
Because pandas counts `nan` & `null` values in `hasnans`

Reply:
> Return True if there are any NaNs or nulls.

If we're checking both nulls and NaNs then the function name should be `has_null_or_nan` instead. Calling `has_nans` but the output is `true` if having nulls is very misleading.

Reply:
It is an established naming convention for pandas, for example, `isna`, `isnull` all account for `nan` & `null` values. We will need to have `hasnan` for parity with pandas similar to the way we do for `isna` & `isnull`. I can add an alias `has_null_or_nan` if you want, but that'll raise the question for what about the rest of the APIs?

Reply:
Keeping the names on par with pandas this way would be really annoying. But I don't want to mess up with this so I'm fine with the name for now :(